core/tests/protractor/featureGatingFlow.js

```diff
@@ -37,41 +47,44 @@ describe('Feature Gating Flow', function() {
 
   beforeAll(async function() {
     adminPage = new AdminPage.AdminPage();
+
+    await users.createAndLoginCurriculumAdminUser(
+      ADMIN_USER1_EMAIL, ADMIN_USERNAME1);
+    await users.logout();
+
+    await users.createAndLoginCurriculumAdminUser(
+      ADMIN_USER2_EMAIL, ADMIN_USERNAME2);
+    await users.logout();
   });
 
   afterEach(async function() {
     await general.checkForConsoleErrors([]);
   });
 
   afterAll(async function() {
-    await users.createAndLoginCurriculumAdminUser(
-      'admin1@featureGatingFlow.com', 'featuregating1');
+    await users.login(ADMIN_USER1_EMAIL, true);
 
     await adminPage.getFeaturesTab();
     var dummy = await adminPage.getDummyFeatureElement();
-    await adminPage.enableFeatureForDev(dummy);
```

Comment:
This was removed because it was not in the original implementation of the e2e test: https://github.com/oppia/oppia/blob/d4ac6ca6ab09df6a4fff0fe4844460e79765aada/core/tests/protractor/featureGatingFlow.js